### DISCUSSION TITLE
[Feature Request] Sound

### DISCUSSION THREAD

**Discussion by shubhamnegi1234** [Voltstro-Studios/UnityWebBrowser#272] [Requests]
### What platform are you experiencing this issue on?

Windows

### What architecture is your platform?

64-Bit

### What version of UWB are you using?

Latest VoltstroUPM Release

### What Unity version are you running?

2021.3.21f1

### Describe what the issue you are experiencing is.

Here is the context, i deployed a webpage and played a YouTube video, and start walking around the game here is the issue, i want to turn off the sound of youtube i don't know where i can access that, I disabled all of the objects in a scene still it was playing.

### Provide reproducible steps for this issue.

1. Deployed UWB
2. Play video or audio over r the internet.
3. how to turn off the sound without pausing or how to mute it from unity

### Any additional info you like to provide?

kindly

**Comment by Voltstro** [comment]
Currently, UWB Core has nothing to do with sound. The engine process it self is what is playing the audio.

You can see the process playing audio (and control it) via the audio mixer in the system (seems to be labeled `Chromium` on Linux, on Windows I believe it comes up as `UnityWebBrowser.Engine.Cef`).
![image](https://github.com/Voltstro-Studios/UnityWebBrowser/assets/45032877/a7461548-da5f-4fa7-bf6f-95c88cc75ecb)

Obviously sound controls in UWB would need to be added. I really don't know what sound APIs CEF provides, research on that would need to be done, and whether sound can be controlled directly, or if it would need to piped back to Unity and played though Unity's audio source or something like that.

**Comment by Voltstro** [comment]
Converted to discussion. Feature Request.
